.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Pull Request
+### Description
+Describe your changes
+### Ticket
+Issue ticket number and link
```

Comment:
I'm not sure if it make sense to put here the link on our own close JIRA board.
Instead of it I suggest putting there all descriptions from the JIRA ticket. including screenshots.
the PR will look like a copy of the JIRA ticket.